save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -163,35 +160,29 @@ data class WarnPluginConfig(
         }
     }
 
+    /**
+     * Substitute placeholders in `this.execFlags` with values from provided arguments
+     *
+     * @param extraFlags [ExtraFlags] to be inserted into `execFlags`
+     * @param fileNames file name or names, that need to be inserted into `execFlags`
+     * @return `this.execFlags` with resolved placeholders
+     */
+    fun resolvePlaceholdersFrom(extraFlags: ExtraFlags, fileNames: String): String =
```

Comment:
Why is this function duplicated in fix and warn plugins? Is there any difference in logic?

Reply:
Fixed